test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/CleanProjectIntegrationTest.cs

```diff
@@ -12,9 +12,11 @@ public class CleanProjectIntegrationTest : MSBuildIntegrationTestBase
         [InitializeTestProject("SimpleMvc")]
         public async Task CleanProject_RunBuild()
         {
-            var result = await DotnetMSBuild("Restore;Build"); // Equivalent to dotnet build
+            var result = await DotnetMSBuild("Restore;Build", "/p:RazorCompileOnBuild=true");
 
             Assert.BuildPassed(result);
+            Assert.FileExists(Project, @"bin\Debug\netcoreapp2.0\SimpleMvc.dll");
```

Comment:
On the CI do we always build tests in debug? Also, is FileExists case insensitive?

Reply:
We're launching the build, so it's whatever we want.